.github/workflows/cibuildwheels.yml

```diff
@@ -1,34 +1,150 @@
+#          CIBW_BEFORE_BUILD: python -m pip install -r requirements.txt
+#          CIBW_BEFORE_TEST: python -m pip install pytest numpy
+#          CIBW_TEST_COMMAND: pytest {project}
+#          CIBW_BUILD_VERBOSITY: 1
+#
+#      - name: Build wheels (Linux / macOS)
+#        if: runner.os != 'Windows'
+#        run: |
+#          python -m pip install cibuildwheel
+#          python -m cibuildwheel --output-dir wheelhouse
+#        env:
+#          CIBW_BUILD: 'cp37-* cp38-* cp39-*'
+#          CIBW_BEFORE_BUILD: python -m pip install -r requirements.txt
+#          CIBW_BEFORE_TEST: |
+#            pip install pytest
+#            pip install -r requirements.txt
+#          CIBW_TEST_COMMAND: pytest {project}
+#          CIBW_BUILD_VERBOSITY: 1
+#
+#      - name: Upload wheels
+#        uses: actions/upload-artifact@v2
+#        with:
+#          path: ./wheelhouse/*.whl
+
+
+
+
+  build_sdist:
+    name: Build SDist
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python-version: [3.7, 3.8]
     steps:
-    - uses: actions/checkout@v2
-      with:
-        submodules: 'recursive'
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
-      with:
-        python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install flake8 pytest
-        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
-    - name: Build
-      run: |
-        python setup.py build_ext --inplace
-
-    - name: Lint with flake8
-      run: |
-        # stop the build if there are Python syntax errors or undefined names
-        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
-    - name: Test with pytest
-      run: |
-        PYTHONPATH=. pytest
+      - uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+
+      - uses: actions/setup-python@v2
+        name: Setup Python ${{ matrix.python-version }}
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pytest
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+
+      - name: Build SDist
+        run: |
+          python setup.py sdist
+
+      - uses: actions/upload-artifact@v2
+        with:
+          path: dist/*.tar.gz
+
+      - name: Build extension from SDist package
+        run: |
+          cd ./dist
+          tar -xzf blosc2-*.tar.gz
+          cd ./blosc2-*/
+          python setup.py build_ext --inplace
+
+      - name: Test with pytest
+        run: |
+          cd ./dist/blosc2-*/
+          PYTHONPATH=. pytest
+
+
+
+#
+#  upload_pypi:
```

Comment:
uncomment this section to upload the sdist

Reply:
Done!